examples/node-app-mssql/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:10
+WORKDIR /tmp/
+
+COPY . .
+RUN npm install
```

Comment:
The [example](https://github.com/leapfrogtechnology/sync-db/tree/master/examples/node-app-mssql) uses `yarn`, please change this to `yarn`